tests/wicked/advanced_ref.pm

```diff
@@ -34,30 +34,41 @@ sub create_tunnel_with_commands {
 sub run {
     my ($self) = @_;
 
-    record_info('Test 1', 'Create a gre interface with IP commands');
+    record_info('Test 1', 'Create a gre interface from legacy ifcfg files');
```

Comment:
hm I didn't notice this. why don't it contains the same test names as suite?

Reply:
Now it does, it should be the same test name, even though REF does it with ip commands.